Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/ButtonSamplePage.xaml

```diff
@@ -4,75 +4,139 @@
 	  xmlns:local="using:Uno.Gallery"
 	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:um="using:Uno.Material"
 	  xmlns:smtx="using:ShowMeTheXAML"
 	  xmlns:todo="what should be done"
 	  mc:Ignorable="d todo">
 
-	<Grid Background="{StaticResource MaterialBackgroundBrush}">
+	<Grid Background="{StaticResource BackgroundBrush}">
 		<local:SamplePageLayout>
 			<local:SamplePageLayout.MaterialTemplate>
 				<DataTemplate>
 					<StackPanel>
 
-						<!-- Contained Buttons -->
-						<smtx:XamlDisplay UniqueKey="ButtonSamplePage_MaterialContained"
-										  smtx:XamlDisplayExtensions.Header="Contained Buttons"
-										  smtx:XamlDisplayExtensions.Description="Contained buttons are high-emphasis, distinguished by their use of elevation and fill. They contain actions that are primary to your app."
+						<!-- Description from: https://m3.material.io/components/buttons/overview -->
+
+						<!-- Elevated Buttons -->
+						<smtx:XamlDisplay UniqueKey="ButtonSamplePage_MaterialElevated"
+										  smtx:XamlDisplayExtensions.Header="Elevated Button"
+										  smtx:XamlDisplayExtensions.Description="Elevated buttons are essentially filled tonal buttons with a shadow. To prevent shadow creep, only use them when absolutely necessary, such as when the button requires visual separation from a patterned background."
 										  smtx:XamlDisplayExtensions.IgnorePath="XamlDisplay\StackPanel">
 							<StackPanel Spacing="20">
 
-								<Button Content="CONTAINED"
-										Style="{StaticResource MaterialContainedButtonStyle}" />
+								<Button Content="ELEVATED" Style="{StaticResource ElevatedButtonStyle}" />
+								<Button Content="ICON ELEVATED" Style="{StaticResource ElevatedButtonStyle}">
```

Comment:
Should we be keeping the Material prefix for all these changes? Once Cupertino and Fluent are updated to the new naming system, this could cause problems, no?